live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -11,33 +11,40 @@ const StyledWelcomeSectionContainer = styled.div`
   @media ${max.tablet} {
     flex-direction: column;
     align-items: center;
+    width: 80%;
```

Comment:
Could you please remove this line? We want it to be almost the whole width and 90% is already being applied elsewhere

Reply:
it didn't work correctly without that. i can switch it to 90%

Reply:
hmm could you please send a screen recording of what it looks like on your end without the width: 80%?

Reply:
https://user-images.githubusercontent.com/59738880/214676886-6c9a5044-d776-4b06-8b8d-26a5a31168e7.mov

The only thing it should change is like this - what happens when you do it?